fileworker.py

```diff
@@ -0,0 +1,39 @@
+from handlers import TxtHandler, JsonHandler
+
+
+class FileWorker:
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.file_types = {'json': JsonHandler, 'txt': TxtHandler}
+
+    def file_extension(self):
+        if self.file_path.endswith('json'):

```

Comment:
ahh .. no, you made dict to not use `if else` operator 😕
Try to find way to get file extension..

Reply:
just imagine if need to handle thousands types of file..